src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -61,6 +63,15 @@ public function getConfigTreeBuilder()
                 ->scalarNode('name_converter')->defaultNull()->info('Specify a name converter to use.')->end()
                 ->booleanNode('enable_fos_user')->defaultValue(false)->info('Enable the FOSUserBundle integration.')->end()
                 ->booleanNode('enable_nelmio_api_doc')->defaultTrue()->info('Enable the Nelmio Api doc integration.')->end()
+                ->scalarNode('db_driver')
```

Comment:
Or better, flags like for FOSUser and NelmioApiDoc: enable_doctrine_orm, enable_doctrine_mongodb_odm.